src/Compilers/CSharp/Test/Symbol/Symbols/MissingSpecialMember.cs

```diff
@@ -643,5 +662,18 @@ public void AllWellKnownTypeMembers()
                 Assert.NotNull(symbol);
             }
         }
+
+        [Fact]
+        public void FirstTypeAfterExtensionMarker()
+        {
+            var e = WellKnownType.System_Runtime_CompilerServices_ValueTuple_T1_T2_T3_T4_T5_T6_T7_TRest.ExtensionByte();
+            Assert.Equal(1, e);
```

Comment:
I am not sure if there is a value in this test, values in an enum are auto incremented. We just need to test that WellKnownType.ExtendedType == 255, which can be done with an assert in static constructor of WellKnownTypes.